Package.swift

```diff
@@ -692,7 +692,7 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         // The 'swift-argument-parser' version declared here must match that
         // used by 'swift-driver' and 'sourcekit-lsp'. Please coordinate
         // dependency version changes here with those projects.
-        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "1.1.4")),
+        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "1.2.0")),
```

Comment:
It's probably preferable to fix that version without the regression as the lowest we allow, but also we could allow 1.3.0 and higher, assuming SAP maintains SemVer compatibility.
```suggestion
        .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.2.2"),
```